docs/modules/ROOT/pages/implementations/prometheus.adoc

```diff
@@ -175,3 +207,52 @@ Actually, the `PrometheusMeterRegistry` has a shortcut for this, so you can do t
 ----
 registry.throwExceptionOnRegistrationFailure();
 ----
+
+== Exemplars
+
+Exemplars are metadata that you can attach to the value of your time series. They can reference reference data outside of your metrics. A common use case is storing tracing information (`traceId`, `spanId`). Exemplars are not tags/dimensions (labels in Prometheus terminology), they will not increase cardinality since they belong to the values of the time series.
```

Comment:
```suggestion
Exemplars are metadata that you can attach to the value of your time series. They can reference data outside of your metrics. A common use case is storing tracing information (`traceId`, `spanId`). Exemplars are not tags/dimensions (labels in Prometheus terminology), they will not increase cardinality since they belong to the values of the time series.
```